docs/preview/features/powershell/arm.md

```diff
@@ -7,6 +7,7 @@ layout: default
 
 This module provides the following capabilities:
 - [Injecting content into an ARM template](#injecting-content-into-an-arm-template)
+- [Removing resource locks within an Azure Resource Group](#removing-resource-locks-within-an-azure-resource-group)
```

Comment:
This line can be removed from this document.

Reply:
Woops, thanks!